modules/openapi-generator/src/test/java/org/openapitools/codegen/testutils/AssertFile.java

```diff
@@ -84,9 +84,10 @@ public FileVisitResult preVisitDirectory(Path expectedDir, BasicFileAttributes a
                         Arrays.sort(actual);
                     }
 
-                    assertEquals(expected,
-                            actual,
-                            String.format(Locale.ROOT, "Directory content of '%s' and '%s' differ.", expectedDir, actualDir));
+                    // Use AssertJ as it also prints a diff
+                    assertThat(actual)
+                            .describedAs(String.format(Locale.ROOT, "Directory content of '%s' and '%s' differ.", expectedDir, actualDir))
+                            .containsExactly(expected);
```

Comment:
is this change related to this PR?

Reply:
Yes and no...
If there are additional or missing files, the old implementation simply does not print /which/ file is missing/surplus but just that "files differ".
One evening, I spent a *lot* of time and nerves searching for a missing file :(

AssertJ gives this information out of the box (assertion behavior otherwise is the same as before)

Removing is no problem (you decide).
In case of removal: if you see just a slight chance that this change might get incorporated, I'd be glad to open another PR just for this because it is just soooo annoying :)

Reply:
Since I have already reviewed it, lets keep it in this PR.

For future PRs, I am happy to review them, and I am even happier if they are small, so I can look at them whenever I have some spare minutes ;)